swagger_to_sdk_config_track2.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "https://raw.githubusercontent.com/lmazuel/swagger-to-sdk/master/swagger_to_sdk_config.schema.json",
+  "meta": {
+    "autorest_options": {
+      "csharp": "",
+      "track2": "",
+      "reflect-api-versions": "",
+      "use": "https://github.com/Azure/autorest.csharp/releases/download/3.0.0-dev.20200617.1/autorest-csharp-v3-3.0.0-dev.20200617.1.tgz",
```

Comment:
@NullMDR is the intent to always have a direct link to code generator like this?

@pakrym do you have plans to publish the code generator to npmjs?

Reply:
Considering azure-sdk-for-net is the only supported consumer. No intent right now.

Reply:
@pakrym Feel free to send our your config PR. I don't know much about the autorest option required for .NET track2.

Reply:
>Would be nice to avoid the version duplication between here and targets file.

@pakrym initially we should just update our AutoRest auto-PR scripts update both files at the same time. We can definitely try to figure out a way to not repeat ourselves as we on-board and see how all the pieces fit together.

Reply:
How is this values used? Is integration invoking out GenerateCode target or uses it's own command line? If it uses our target why do we need this value, if not - it should because having 2 ways to generate the same code would only cause issues in the future.

What about other versions that we pin? Does automation take them into account?

Reply:
The automation directly calls autorest as it is a general set of tools across languages. It isn't calling the GenerateCode target in msbuild. Although I'm not familiar enough yet to know whether or not we can call a completely separate command but it does seem to me that we should support calling autorest directly.

Reply:
In the repo we pin both autorest version, autorest core version and C# generator version if automation would use different versions it might result in different generation results.

Does the automation use autorest.md files from the repo as the input?

Reply:
That is a question for @NullMDR as I'm not entirely sure if that was allowed yet. I do agree we should try to figure out a common way between running this in the repo and running it in automation. 

From a quick look at https://dev.azure.com/devdiv/DevDiv/_git/openapi-sdk-automation?path=%2Fsrc%2FsdkRepository.ts it seems like the automation is making direct calls to the autorest library instead of the command line. However I'm still trying to understand how the full system works.

Reply:
@pakrym SDK Automation call autorest commandline directly with the parameter provided. What do you mean by "use autorest.md files from the repo as the input"? Is this some behavior of autorest?

Reply:
I believe he is referring to issue https://github.com/Azure/azure-rest-api-specs/issues/8955.